daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -61,11 +62,21 @@ func New(info logger.Info) (logger.Logger, error) {
 		}
 	}
 
-	var extra []byte
 	attrs, err := info.ExtraAttributes(nil)
 	if err != nil {
 		return nil, err
 	}
+
+	// no default template. only use a tag if the user asked for it
+	tag, err := loggerutils.ParseLogTag(info, "")
```

Comment:
Why is there no default template? The first 12 bytes of the containerID should be the default according to [our docs] (https://docs.docker.com/engine/admin/logging/log_tags/). Also, other logging drivers have this default.